package.json

```diff
@@ -32,15 +32,15 @@
   },
   "scripts": {
     "clean": "git clean -x -f --exclude=packages/*/.env --exclude=cardstack.code-workspace && lerna run clean",
-    "postinstall": "cd ./packages/cardpay-sdk && yarn codegen; cd -",
+    "postinstall": "cd ./packages/cardpay-sdk && yarn codegen",
```

Comment:
Consider using the [cwd option](https://stackoverflow.com/questions/46891622/run-yarn-in-a-different-path):

```
"postinstall": "yarn --cwd packages/cardpay-sdk codegen",
```